### ISSUE TITLE
Would it be possible to support Carthage Binary?

### ISSUE THREAD

**Issue by bsarrazin** [RxSwiftCommunity/RxKeyboard#47]
So I've been digging around a bit around the issues raised in #45, see https://github.com/Carthage/Carthage/issues/2245
Carthage as a binary-only support (documented [here](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#binary-only-frameworks)).

This means that users of RxKeyboard could simply add the following in their `Cartfile`:
```ruby
binary "https://github.com/RxSwiftCommunity/RxKeyboard/releases/Carthage.json"
```

Above is the proper, documented and recommended way to support binary-only libraries.
This would mean that we need to add a JSON file with the information.

Thoughts?

**Comment by bsarrazin** [comment]
Created pull request above ^^
Tested and working.

**Comment by devxoul** [comment]
Thanks! I merged #48 :)
